packages/vue/src/index.ts

```diff
@@ -2,24 +2,24 @@ import type { VNode } from 'vue'
 import { createTextVNode, h } from 'vue'
 import { RouterLink } from 'vue-router'
 
-import type { SbRichTextNode, SbRichTextNodeResolver, SbRichTextOptions } from '@storyblok/richtext'
+import type { StoryblokRichTextNode, StoryblokRichTextNodeResolver, StoryblokRichTextOptions } from '@storyblok/richtext'
 import { BlockTypes, MarkTypes, richTextResolver } from '@storyblok/richtext'
 import { StoryblokComponent } from '@storyblok/vue'
-import SbRichText from './components/SbRichText.vue'
+import StoryblokRichText from './components/StoryblokRichText.vue'
```

Comment:
@alvarosabu here we're importing from `./components/StoryblokRichText.vue` but seems like the file wasn't renamed and is still `./components/SbRichText.vue`.

Same for the one in `react/src/SbRichText.tsx`